org.nomacs.ImageLounge.json

```diff
@@ -0,0 +1,134 @@
+    ],
+    "build-options": {
+        "arch": {
+            "x86-64": {
+                "no-debuginfo": true,
+                "strip": true
+            },
+            "aarch64": {
+                "no-debuginfo": false,
+                "strip": false
+            },
+            "arm": {
+                "no-debuginfo": false,
+                "strip": false
+            }
+        }
+    },
+    "modules": [
+        {
+            "name": "exiv2",
+            "buildsystem": "cmake",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.exiv2.org/builds/exiv2-0.27.2-Source.tar.gz",
+                    "sha256": "2652f56b912711327baff6dc0c90960818211cf7ab79bb5e1eb59320b78d153f"
+                }
+            ]
+        },
+        {
+            "name": "opencv",
+            "buildsystem": "cmake",
+            "builddir": true,
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/opencv/opencv/archive/4.1.0.tar.gz",
+                    "sha256": "8f6e4ab393d81d72caae6e78bd0fd6956117ec9f006fba55fcdb88caf62989b7"
+                }
+            ]
+        },
+        {
+            "name": "libraw",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.libraw.org/data/LibRaw-0.19.5.tar.gz",
+                    "sha512": "4560045f75e6d2ab0d1d8686075f3a0e26a5d7ce693b48508110a2c31d19055d58983c24852da0abb64fa90db5e20f24b87aa7537ed04d958c38c8b265a7e826"
+                }
+            ]
+        },
+        {
+            "name": "quazip",
+            "buildsystem": "cmake",
+            "builddir": true,
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/stachenov/quazip/archive/v0.8.1.tar.gz",
+                    "sha512": "d65dab16c2cbc4d8bb23dae5b2fdff5ef787aaff2653b8e75c3fc8ba1701f5568a52bdde39009f83c5069a9fe3331cd1a49cf965822c4180e88bbc341faa2bfb"
+                },
+                {
+                    "type": "patch",
+                    "path": "quazip.patch",
+                    "strip-components": 1
+                }
+            ]
+        },
+        {
+            "name": "nomacs",
+            "buildsystem": "cmake",
+            "config-opts": [
+                "-DUSE_SYSTEM_QUAZIP=ON"
+            ],
+            "subdir": "ImageLounge",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/nomacs/nomacs/archive/3.14.tar.gz",
+                    "sha512": "09d4c222f421661a09b7dfd5d2d162a5bb773dd9b262563dfe9f8af4fafd5d653892a1c42993154cf362fb69668bc4bac42da3a0b32b04b421513eca1ae54ec3"
+                }
+            ]
+        },
+        {
+            "name": "nomacs-desktop-integration",
```

Comment:
Since you are developer: Put these files into the source tree.

cmake example:
https://github.com/obsproject/obs-studio/tree/master/UI/xdg-data

Reply:
cool, thanks for the sample!